includes/validation/class-amp-validation-manager.php

```diff
@@ -1975,32 +1956,72 @@ public static function get_validate_url_error_message( $error_code, $error_messa
 			);
 		}
 
+		$implode_non_empty_strings_with_spaces = static function ( $strings ) {
+			return implode( ' ', array_filter( $strings ) );
+		};
+
 		switch ( $error_code ) {
 			case 'http_request_failed':
-				return __( 'Failed to fetch URL to validate.', 'amp' ) . $error_message . $site_health_message . $support_forum_message;
+				return $implode_non_empty_strings_with_spaces(
+					[
+						esc_html__( 'Failed to fetch URL to validate.', 'amp' ),
+						esc_html( $error_message ),
+						$site_health_message,
+						$support_forum_message,
+					]
+				);
 			case 'white_screen_of_death':
-				return __( 'Unable to validate URL. Encountered a white screen of death likely due to a PHP fatal error.', 'amp' ) . $error_message . $check_error_log . $support_forum_message;
+				return $implode_non_empty_strings_with_spaces(
+					[
+						esc_html__( 'Unable to validate URL. A white screen of death was encountered which is likely due to a PHP fatal error.', 'amp' ),
+						esc_html( $error_message ),
+						$check_error_log,
+						$support_forum_message,
+					]
+				);
 			case '404':
-				return __( 'The fetched URL was not found. It may have been deleted. If so, you can trash this.', 'amp' ) . $error_message . $support_forum_message;
+				return $implode_non_empty_strings_with_spaces(
+					[
+						esc_html__( 'The fetched URL was not found. It may have been deleted. If so, you can trash this.', 'amp' ),
```

Comment:
You're right. Trash isn't right. Actually the terminology used is “forget”:

https://github.com/ampproject/amp-wp/blob/e6f9dc3587954e40c0c21bc66ccfa88f99aaa1b3/includes/validation/class-amp-validated-url-post-type.php#L2281-L2292

So how about:

```suggestion
						esc_html__( 'The fetched URL was not found. It may have been deleted. If so, you can forget this validated URL.', 'amp' ),
```

Reply:
`forget` is what I had originally in my previous comment, but it sounded confusing to me. If `forget` is going to be used though, I would suggest making it a link and adding a perhaps adding a `title` attribute to provide context.

Reply:
In reality, this error message is very rarely shown. It can only happen if someone has a validated URL post open in one tab, then someone forgets the URL in another tab, and then tries to re-check the URL that had been forgotten. As such, I don't think we need to worry too much about tailoring the error message.